projects/Mallard/ios/Mallard.xcodeproj/project.pbxproj

```diff
@@ -248,6 +249,7 @@
 		13B07FAE1A68108700A75B9A /* Mallard */ = {
 			isa = PBXGroup;
 			children = (
+				9082A5BA243E82B500214C51 /* GuardianDailyDebug.entitlements */,
```

Comment:
I'm wondering whether this change is a hangover from when we were using a different bundle id on this branch

Reply:
maybe, its a generated file. We could delete the file and see if Xcode regenerates a clean version.